lib/polyfill/vttcue.js

```diff
@@ -0,0 +1,111 @@
+/**
+ * @license
+ * Copyright 2016 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+goog.provide('shaka.polyfill.VTTCue');
+
+goog.require('shaka.log');
+goog.require('shaka.polyfill.register');
+
+
+/**
+ * @namespace shaka.polyfill.VTTCue
+ *
+ * @summary A polyfill to provide VTTCue.
+ */
+
+
+/**
+ * Install the polyfill if needed.
+ */
+shaka.polyfill.VTTCue.install = function() {
+  if (window.VTTCue) {
+    shaka.log.info('Using native VTTCue.');
+    return;
+  }
+
+  if (!window.TextTrackCue) {
+    shaka.log.info('VTTCue not available.');
```

Comment:
This shouldn't happen, since all the browsers we support have either VTTCue or TextTrackCue.  So we probably don't need a no-op polyfill for this case.

But if this were to happen, we would throw at new VTTCue in TextEngine, so this message should at least be upgraded to shaka.log.error.